tests/utils/allure.js

```diff
@@ -0,0 +1,29 @@
+const allure = require('allure-commandline');
+
+const generateReport = () => {
+  const reportError = new Error('Could not generate Allure report');
+  const timeoutError = new Error('Timeout generating report');
```

Comment:
Are you generating these upfront to capture the stacktrace or something? Otherwise it'd be better to do them on demand, but I can't remember if modern node carries the stack into the promise or not...

Reply:
I just moved some code that already existed, to have these run for the upgrade e2e test, which could not call the generic teardown because we have different containers.